read.js

```diff
@@ -10,6 +10,9 @@ async function main() {
     /**
      * The Mongo Client you will use to interact with your database
      * See https://mongodb.github.io/node-mongodb-native/3.6/api/MongoClient.html for more details
+     * In case: '[MONGODB DRIVER] Warning: Current Server Discovery and Monitoring engine is deprecated...'
+     * pass option { useUnifiedTopology: true } to the MongoClient constructor.
+     * const client =  new MongoClient(uri, {useUnifiedTopology: true})
```

Comment:
I would use this one as the default, no?

Reply:
The Node driver team asked me not to include it in the examples as default as the new 4.0 driver takes care of this for you.